web/src/pages/Administration/UserManagement/UserList.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import { Grid, Typography, Button, Divider, Tabs, Tab, Box } from '@mui/material';
+import SimpleTable from '../../../components/table/simpleTable';
+import { InternalUserTableData } from './Data';
+import Filters from '../../../components/Filter/filter';
+import { COMPONENTS } from '../../../utils/constants';
+import './UserList.scss';
+
+const TabPanel = (props) => {
+  const { children, value, index, ...other } = props;
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`scrollable-auto-tabpanel-${index}`}
+      aria-labelledby={`scrollable-auto-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+};
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`
+  };
+}
+
+function UserList() {
+  const [value, setValue] = useState(0);
+  const [internalUserData, setInternalUserData] = useState(InternalUserTableData);
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  const columnForInternalUser = [
+    { field: 'id', header: 'ID', sortable: true, filter: true },
```

Comment:
Headers should be Multilingual so pass the key name and define same in translations files